celements-xwiki-core/src/main/java/com/xpn/xwiki/objects/BaseCollection.java

```diff
@@ -263,7 +263,7 @@ public BaseClass getXClass(XWikiContext context) {
       try {
         baseClass = context.getWiki().getXClass(classReference, context);
       } catch (Exception e) {
-        logger.error("Failed to get class [" + classReference + "]", e);
+        logger.error("getXClass - failed to get class [" + classReference + "]", e);
```

Comment:
```suggestion
        logger.error("getXClass - failed to get class [{}]", classReference, e);
```
why not using slf4j message formatting?